static/js/directives/chat.js

```diff
@@ -329,6 +356,14 @@ define(['jquery', 'underscore', 'text!partials/chat.html', 'text!partials/chatro
 								console.error("Failed to receive geolocation", err);
 							});
 						};
+						subscope.showFingerprint = function() {
+							if (subscope.fingerprint) {
+								// TODO(fancycode): Show a nicer notification.
```

Comment:
What would be a nicer notification? I guess it should avoid a modal popup and rather show this somewhere inside the chat frame.